burn-book/src/custom-training-loop.md

```diff
@@ -101,11 +98,8 @@ pub fn run<B: ADBackend>(device: B::Device) {
             let accuracy = accuracy(output, batch.targets);
 
             println!(
-                "[Valid - Epoch {} - Iteration {}] Loss {} | Accuracy {}",
-                iteration,
-                epoch,
+                "[Valid - Epoch {epoch} - Iteration {iteration}] Loss {} | Accuracy {accuracy}",
```

Comment:
I think we should have the same formatting for both training and validation prints. There is no precision settings in the validation output `{:.3}`.

Reply:
I mainly made my changes in an attempt to still keep the original styling of the code section, but IMO it could be improved. I assume you want to use the same style from the training prints, right? (I.E, precision of 3 digits and the % sign)

Personally I am not really a fan of just printing like this in the code, and I think the training TUI would make a lot more sense here, but I am also not sure how difficult it would be to actually add this to the code.

Another thing is that perhaps you should consider moving this chapter to another part of the book, as it's really less about a custom training loop and more about how a training loop in Burn works in general (which is still something that is very useful to know, it should perhaps just be presented in a different way).

Reply:
Yes the goal is to have the same print statement for training and validation. Do you have some propositions in presenting this book section a bit differently?